cape_privacy/policy/policy_test.py

```diff
@@ -251,3 +251,14 @@ def test_secret_in_named_transform():
     new_df = policy_lib.apply_policy(p, df)
 
     pdt.assert_frame_equal(new_df, df)
+
+
+def test_reverse_helper():
```

Comment:
could you write a test that applies a policy to a df, gets a reverse policy, applies it to your tokenized df, and assert you get the original back?